pkg/apis/ceph.rook.io/v1/scc.go

```diff
@@ -43,7 +43,7 @@ func NewSecurityContextConstraints(name, namespace string) *secv1.SecurityContex
 		AllowHostNetwork:         false,
 		AllowHostPorts:           false,
 		AllowedCapabilities:      []corev1.Capability{"MKNOD"},
-		RequiredDropCapabilities: []corev1.Capability{},
+		RequiredDropCapabilities: []corev1.Capability{"ALL"},
```

Comment:
@travisn I don't have any uses for this file/function anywhere else in the code. Is it safe to remove?

Reply:
Actually, this is the rook API,
Which can be called from other Projects where they wanted to import this(for ex: openshift),

Reply:
Correct, not used by rook, but other projects that reference rook